collect_app/src/main/res/drawable/splash_screen_icon.xml

```diff
@@ -0,0 +1,314 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I had to add a vector version of the icon to display it in the splash screen. I think it looks pretty good (similar to the original one) but let me know if you have a better one.

Reply:
Works for me. I don't have a different one.